vital/range/defs.h

```diff
@@ -168,26 +168,28 @@ struct function_detail< ReturnType ( Object::* )( ArgsType... ) const >
 
 // ----------------------------------------------------------------------------
 namespace range_detail {
-  using namespace std;
 
-  template < typename Range >
-  struct range_helper
+using namespace std;
```

Comment:
I know this has been here, but I really wish we didn't use 'using namespace std;', especially in such a common header. Seems rather polluting, no?

Reply:
I'm not a big fan of `using namespace xxx`

Reply:
Eh... it's isolated in `namespace range_detail`. I can look at replacing this with `using std::begin; using std::end;` though...